tests/test_user.py

```diff
@@ -110,6 +110,19 @@ def test_unique_email(self):
         results = response.json
         assert results['unique'] is True
 
+    def test_unique_hidden_email(self):
+        # hidden emails, such as masked invited user create collisions
+        email = 'fake@email.com'
+        fake_user = self.add_user(username=email)
+        fake_user.mask_email()
+
+        self.login()
+        request = '/api/unique_email?email={}'.format(urllib.parse.quote(
```

Comment:
Nitpick: could you use `query_string` kwarg to `self.client.get` to build the URL instead of string formatting?

It's a little more work to setup, but is much easier to maintain
eg:
```
response = self.client.get(
    '/api/unique_email', 
    query_string={'email': urllib.parse.quote(email)},
)
```

Reply:
Yes, just a cut and paste detail.  Perhaps you'd like to add an issue to go through all of our tests and make this change?  I agree with your nitpick - lets sweep through and get 'em all.